components/atoms/ToggleGroup/toggle-group.tsx

```diff
@@ -0,0 +1,59 @@
+import * as GroupToggle from "@radix-ui/react-toggle-group";
+import { useState } from "react";
+import ToggleGroupItem from "../ToggleGroupItem/toggle-group-item";
+
+interface ToggleGroupProps {
+    children: React.ReactNode,
+    /** Allows the user to deselect an option, resulting in none being selected. */
+    allowNone?: boolean,
```

Comment:
A descriptive name is much cleaner than many lines of comments.

Reply:
Got suggestions for names?
Those also aren't normal comments, they are JSDoc comments/description that will show up for each of those property when hovered on, whenever a developer tries to implement them anywhere

Reply:
![image](https://user-images.githubusercontent.com/52013393/208983855-33c5c3e5-8e14-49c1-9d30-9abf4d63d063.png)